cmd/crc/cmd/config/config.go

```diff
@@ -22,6 +23,8 @@ var (
 	// Start command settings in config
 	VMDriver = createSetting("vm-driver", nil, []validationFnType{validations.ValidateDriver})
 	Bundle   = createSetting("bundle", nil, nil)
+	CPUs     = createSetting("cpus", constants.DefaultCPUs, []validationFnType{validations.ValidateCPUs})
+	Memory   = createSetting("memory", nil, []validationFnType{validations.ValidateMemory})
```

Comment:
We have a default for memory...

we currently take the default from the bundle...  but that would be the same for the CPUs, so why different here?